tools/make/ffi.toml

```diff
@@ -71,7 +71,7 @@ command = "cargo"
 args = ["check", "--package", "icu_freertos",
         "--target", "thumbv8m.main-none-eabihf",
         "--no-default-features", "--features=wearos",
-        "-Zbuild-std=std,panic_abort", "-Zbuild-std-features=panic_immediate_abort"]
+        "-Zbuild-std=core,alloc", "-Zbuild-std-features=panic_immediate_abort"]
```

Comment:
this should still be panic_abort, yes?

Reply:
The panic_abort crate is only used by libstd.